src/cache.cpp

```diff
@@ -10,13 +12,19 @@ Cache::Cache(const CACHE_SET &cfg)
     _cache_setting = cfg;
     _Cache_Setup();
 }
-
+Cache::Cache()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'Cache::_has_evicted' is not initialized in the constructor.](https://app.codacy.com/app/jerry0715no14/Cache_sim/pullRequest?prid=4646302)